README.md

```diff
@@ -3,16 +3,22 @@
 This repo holds information about subnet chains that exist on the Avalanche
 network.
 
+## To add your subnet to this list, please make a pull request that adds the following: 
+
 Each chain in the `chains` folder may contain the following files:
-- `chain-information.json`: General information about the chain such as the
-  name, vm configuration, and token configuration.
+- `genesis.json`: To get started, use the
+  [Avalanche CLI](https://github.com/ava-labs/avalanche-cli) to enter your desired configuration parameters and copy over the resulting file using the `export` command.
+- `chain-information-rpc.json`: Minimum information about the chain for a node to be able to connect to your network.
```

Comment:
Would be good to make this more specific as it's not clear from this description what should be included in this file.

Reply:
I included and example in this directory:
`chain-information-rpc.json.example`

Reply:
If this is going to duplicate data from `chain-information.json` would rather just have one file to collect all this info, with better descriptions about what each field is and what it is needed for.

Is there a reason we need this additional file type?

Reply:
I'm not sure that duplicating this information across a "light" and "full" version of the files makes sense.  Personally I would prefer if we just lowered the requirements for what is stored in the full chain-information.json to have minimal required attributes, with the extra data used by the explorer as optional values.  Then we will have a separate filter to control which chains actually have sufficient data for usage in the explorer/glacier/core.